cedar-dafny/validation/thm/model.dfy

```diff
@@ -837,6 +854,23 @@ module validation.thm.model {
     ensures eval.interpretList(es).Err? ==> exists i :: 0 <= i < |es| && eval.interpret(es[i]).Err? && eval.interpret(es[i]).error == eval.interpretList(es).error && (forall j | 0 <= j < i :: eval.interpret(es[j]).Ok?)
   {}
 
+  lemma InterpretListEnsuresOk(eval: Evaluator, es: seq<Expr>)
```

Comment:
Nit: I was wondering if it makes sense if pull out lemmas related to the evaluator to its own files. I think other features could use these lemmas as well.

Reply:
@emina what do you think?

Reply:
Sounds good to me, especially if we have several of these.